### DISCUSSION TITLE
Insecure Request

### DISCUSSION THREAD

**Discussion by MAIQUE100** [TadaSoftware/PyNFe#251] [General]
Bom dia, ao executar o programa, ele retorna o seguinte erro:

C:\Python310\lib\site-packages\urllib3\connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www1.nfe.fazenda.gov.br'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

Solicito ajuda nesse sentido, por favor.

Desde ja agradeço. :)

**Comment by leogregianin** [comment]
> Bom dia, ao executar o programa, ele retorna o seguinte erro:
> 
> C:\Python310\lib\site-packages\urllib3\connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www1.nfe.fazenda.gov.br'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings warnings.warn(
> 
> Solicito ajuda nesse sentido, por favor.
> 
> Desde ja agradeço. :)

Não é bem um "erro", é um warning da própria biblioteca urllib3.

Você pode omitir o warning com `urllib3.disable_warnings()`
